<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfgTrainForm</class>
 <widget class="QWidget" name="CfgTrainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>初始资金:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dbspbBegCapital">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>40000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000.000000000000000</double>
   </property>
   <property name="value">
    <double>50000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dbspbFeeOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>25.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>平仓手续费:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开仓手续费:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dbspbFeeOpen_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="chbox_auto_stop_loss">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>自动止损</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_stop_loss">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbox_auto_stop_profit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>自动止盈</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_stop_profit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbtn_save">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>dbspbFeeOpen</zorder>
  <zorder>label_9</zorder>
  <zorder>label_3</zorder>
  <zorder>dbspbFeeOpen_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>pbtn_cancel</zorder>
  <zorder>pbtn_save</zorder>
  <zorder>dbspbBegCapital</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
